### ISSUE TITLE
mkfs options out of order

### ISSUE THREAD

**Issue by jgreat** [Azure/WALinuxAgent#684]
Versions: 

- Azure Linux Agent Version:2.2.10
- OS: alpine 3.5.2
- mkfs.ext4: mke2fs 1.43.3 (04-Sep-2016)
- mkfs.xfs: mkfs.xfs version 4.5.0

I'm running the agent in an alpine based container.  

I guess mkfs in alpine is picky about the arguments order.  Placing the force flag after the device throws an error, which prevents the resource disk from being mounted. 

```
017/04/29 16:35:48.967782 INFO Azure Linux Agent Version:2.2.10
2017/04/29 16:35:48.968630 INFO OS: alpine 3.5.2
2017/04/29 16:35:48.969190 INFO Python: 3.5.2
2017/04/29 16:35:48.969806 INFO Run daemon
2017/04/29 16:35:48.970377 INFO No RDMA handler exists for distro='' version='3.5.2'
2017/04/29 16:35:48.972637 INFO Activate resource disk
2017/04/29 16:35:48.974758 VERBOSE Run 'mount'
2017/04/29 16:35:48.978003 INFO Examining partition table
2017/04/29 16:35:48.978596 VERBOSE Run 'parted /dev/sdb print'
2017/04/29 16:35:48.986409 INFO GPT not detected, determining filesystem
2017/04/29 16:35:48.987093 VERBOSE Run 'sfdisk --part-type -f /dev/sdb 1 -n'
2017/04/29 16:35:48.989954 INFO sfdisk --part-type -f /dev/sdb 1 -n succeeded
2017/04/29 16:35:48.990559 INFO The partition is formatted with ntfs, updating partition type to 83
2017/04/29 16:35:48.991094 VERBOSE Run 'sfdisk --part-type  /dev/sdb 1 83'
2017/04/29 16:35:49.023551 INFO sfdisk --part-type  /dev/sdb 1 83 succeeded
2017/04/29 16:35:49.024264 VERBOSE Run 'sfdisk -R /dev/sdb'
2017/04/29 16:35:49.027466 VERBOSE Run 'blockdev --rereadpt /dev/sdb'
2017/04/29 16:35:49.033235 INFO Format partition [mkfs.ext4 /dev/sdb1 -F]
2017/04/29 16:35:49.034486 VERBOSE Run 'mkfs.ext4 /dev/sdb1 -F'
2017/04/29 16:35:49.037876 ERROR Command: 'mkfs.ext4 /dev/sdb1 -F'
2017/04/29 16:35:49.038539 ERROR Return code: 1
2017/04/29 16:35:49.039512 ERROR Result: mke2fs 1.43.3 (04-Sep-2016)
mkfs.ext4: invalid blocks '-F' on device '/dev/sdb1'

2017/04/29 16:35:49.040590 INFO Mount resource disk [mount /dev/sdb1 /mnt/resource]
2017/04/29 16:35:49.041095 VERBOSE Run 'mount /dev/sdb1 /mnt/resource'
2017/04/29 16:35:49.119355 WARNING Failed to mount resource disk. Retry mounting after re-reading partition info.
```

**Comment by jgreat** [comment]
PR that fixes the order: #685

**Comment by brendandixon** [comment]
Merged #685.
